src/server.ts

```diff
@@ -0,0 +1,24 @@
+// Import the 'express' module along with 'Request' and 'Response' types from express
+import express, { Request, Response } from "express";
+import cors from "cors";
+// Create an Express application
+const app = express();
+
+// Specify the port number for the server
+//const PORT = process.env.PORT || 3002;
```

Comment:
clean it please, tidy clean code

better not to keep commented code

Reply:
comments deleted